app/lib/mini-course.cjsx

```diff
@@ -153,8 +157,19 @@ module.exports = React.createClass
 
       @props.projectPreferences.update "preferences.minicourses.completed_at.id_#{@props.minicourse.id}": now
       @props.projectPreferences.save()
+
+      @logToGeordi now
```

Comment:
Having a bit of a tougher time testing this on the staged branch because I'm always getting the `else` of this conditional, but it appears to be set up identically to the tutorial instance and should work.